Test/Business_classes/money_tranfer_usecase_test.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from Domain_classes.Account import Account
+from Business_classes.Money_Transfer_Usecase import MoneyTransferUsecase
+from Business_classes.Money_Transfer_Command import MoneyTransferCommand
+
+
+class MoneyTransferUseCaseTest(unittest.TestCase):
+    repo = {
+        "123": Account('123', 'Bartek', 444),
+        "456": Account('456', 'Łukasz', 0),
+        "789": Account('789', 'Michał', 60)
+    }
+
+    def test_one_account_locked(self):
+        # Given
+        self.repo["123"] = Account('123', 'Bartek', 444, '354')
+        usecase = MoneyTransferUsecase(self.repo)
+        # When
+        command = MoneyTransferCommand('123', '456', 222)
+        # Then
+        with self.assertRaises(Exception):
+            usecase.transfer(command)
+
+    def test_one_account_no_money(self):
+        self.repo["123"] = Account("123", "Bartek", 50)
+        self.repo["456"] = Account("456", "Lukasz", 0)
+        usecase = MoneyTransferUsecase(self.repo)
+        command = MoneyTransferCommand("456", "123", 100)
```

Comment:
Czy w tym przypadku nie powinniśmy też okczekiwać wyjątku ?

Reply:
no tu generalnie na dalszych galeziach napisalem tez wlasnie, ze dostajemy wyjatek, ale wspolnie z Łukaszem doszlismy do wniosku, że trzeba najpierw sprawdzic czy money amount jest wieksze od aktualnego stanu konta i uzylismy tam asserFalse, mozna bylo zamiennie tez użyć True, ale jak tak teraz patrze to wciaz chyba pasowaloby dopisac ten Exeption z komunikatem z jakims if'em. To wszystko tez na dalszych galeziach